### PR TITLE
Pull latest translations from Transifex

### PR DESCRIPTION
**Pull request by juho-jaakkola**

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
I fixed a syntax error in one of the language files. Translator had entered funny syntax in the translation in Transifex.

File: languages/nl.php [hunk 1]
```diff
@@ -341,8 +372,8 @@
 	'river:none' => 'Geen activiteit',
 	'river:update' => 'Update van %s',
 	'river:delete' => 'Verwijder deze activiteit',
-	'river:delete:success' => 'Het river-item is verwijderd',
-	'river:delete:fail' => 'Het river-item kon niet worden verwijderd',
+	'river:delete:success' => 'Activity item has been deleted',
+	'river:delete:fail' => 'Activity item could not be deleted',
```
Comment (jdalsem) [inline languages/nl.php]: any idea why this is reverted to english? Dutch translation was present... maybe removed in transifex?

Comment (juho-jaakkola) [reply languages/nl.php]: Yes, seems to be missing from Transifex for some reason.

**Comment by jdalsem** [comment]
good luck upmerging :)

**Pull request merged**